README.md

```diff
@@ -1,4 +1,4 @@
-# The OpenAPI Specification (fka The Swagger Specification)
+# The OpenAPI Specification (originally cloned from The Swagger Specification)
```

Comment:
Technically, the spec repo was transferred, not cloned. This isn't a copy of the Swagger spec, it is an evolution.